src/tests/ui/planner/toolbarHeader.ts

```diff
@@ -22,7 +22,8 @@ export class ToolbarHeader extends BaseElement {
   saveFilterDialog = new BaseElement(this.$('.save-filter-dropdown'));
   saveFilterBtn = new ui.Button(this.saveFilterDialog.$('.save-cq-btn'),'Save');
   closeBtn = new ui.Button(this.$('.cancel-cq-btn'),'Cancel');
-  titleTextInput = new ui.TextInput(this.saveFilterDialog.$('input.form-control'), 'Query Title');
+  titleTextInput = new ui.TextInput(this.saveFilterDialog.$('input.query-title'), 'Query Title');
```

Comment:
@divyanshiGupta why you changed titleTextInput css ? Custom query test is failing because of this. Are you using it anywhere ?

Reply:
@Raunak1203 it might have happened due to a merge conflict. I will change it back.